src/tests/nurse/test_views.py

```diff
@@ -155,43 +155,48 @@ def prescription(user):
 
 @pytest.mark.django_db
 class TestSendEmailToDoctorView:
-    prescription = prescription_data
+    valid_payload = {"additional_info": "This is a valid message."}
+    invalid_payload = {"additional_info": "This message contains <invalid> characters."}
     url = reverse_lazy("send-email-to-doctor", kwargs={"pk": 1})
 
     def test_endpoint(self):
         assert self.url == "/prescription/1/send-email/"
 
     def test_send_email_to_doctor(self, client, prescription):
-        response = client.post(self.url)
+        response = client.post(self.url, self.valid_payload)
         assert response.status_code == status.HTTP_200_OK
         assert response.data == {"message": "Email sent"}
         assert len(mail.outbox) == 1
         assert mail.outbox[0].to == [prescription.email_doctor]
 
+        # Check the rendered email message
+        email = mail.outbox[0]
+        assert "Renouveler ordonnance" in email.subject
+
```

Comment:
Yes the type of things I had in mind, do the same for a few dynamic lines in the body e.g.
- `"</li>Patient : {{ patient_name }}</li>" in email.body`
- `"<li>Date de naissance : {{ patient_birth_date }}</li>" in email.body`
- `<pre>{{ additional_info }}</pre>`
- `<p>{{ nurse_name }}<br>`

You get the idea, we want to make sure things get rendered properly

Reply:
Extra hint:
```
assert email.alternatives[0][1] == "text/html"
html_body = email.alternatives[0][0]
```

Reply:
The problem was caused by differences between the spacing and line feeds in the HTML generated by the email and what the test expected.